inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -240,7 +281,7 @@ export class CanvasGraphService {
 
         this._generateTicks(timeMinMax, spaceAvailable);
 
-        const axisHeight = 100;
+        const axisHeight = this._defaultLineHeight + 30;

```

Comment:
Would be great to have 30 in a const so at least by looking at this line we know what is happening ? Even better as it is almost "static" you could precompute axisHeight when you check for _defaultLineHeight ? (premature optim but there are no small gains :-) )

Reply:
Done!